examples/bri-3/src/bri/transactions/api/transactions.controller.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  Param,
+  Post,
+  Put,
+} from '@nestjs/common';
+import { CommandBus, QueryBus } from '@nestjs/cqrs';
+import { CreateTransactionCommand } from '../capabilities/createTransaction/createTransaction.command';
+import { DeleteTransactionCommand } from '../capabilities/deleteTransaction/deleteTransaction.command';
+import { UpdateTransactionCommand } from '../capabilities/updateTransaction/updateTransaction.command';
+import { CreateTransactionDto } from './dtos/request/createTransaction.dto';
+import { UpdateTransactionDto } from './dtos/request/updateTransaction.dto';
+
+@Controller('transactions')
+export class TransactionController {
+  constructor(private commandBus: CommandBus, private queryBus: QueryBus) {}
+
+  @Post()
+  async createTransaction(
```

Comment:
do we add get methods?

Reply:
As part of the PR that will be opened soon, just wanted to avoid to much functionality in a single PR